rpc/detector/service.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package trivy.detector;
+option go_package = "detector";
+
+service OSDetector {
+    rpc Detect(OSDetectRequest) returns (DetectResponse);
+}
+
+message OSDetectRequest{
+    string os_family = 1;
+    string os_name = 2;
+    repeated Package packages = 3;
+}
+
+message DetectResponse{
+    repeated DetectedVulnerability detected_vulnerabilities= 1;
+}
+
+message Package {
+    string name = 1;
+    string version = 2;
+    string release =3;
+    int32  epoch = 4;
+    string arch = 5;
+    string src_name = 6;
+    string src_version = 7;
+    string src_release = 8;
+    int32 src_epoch = 9;
```

Comment:
What's the difference between epoch and src_epoch? Lets explain it in proto doc / comment

Reply:
I added a comment